### ISSUE TITLE
[TSDoc> Playground]: There is no visual label is defined for the 'Select an editor theme and Select a code sample' combo box.

### ISSUE THREAD

**Issue by vagpt** [microsoft/rushstack-websites#254]
Test Environment:
Version: Version 24H2 (OS Build 26231.5000)
Browser Version: 127.0.2638.2 (Official build) dev (64-bit)
URL: https://tsdoc.org/play/
Screen Reader: Narrator

**Repro Steps:**

1. Open the above URL on the EDGE Dev browser.
2. Press the tab key and navigate to the 'PlayGround' option and press enter key to activate it.
3. Press the tab key and navigate to Select an editor theme and Select a code sample combo box and observe the issue.

**Actual Result:**
No visual label is defined for the Select an editor theme and Select a code sample combo box. When the focus lands on it then screen reader is announcing as 'TSDoc Playground document select a code sample choose a sample combo box collapsed has popup.'
Also, when user saying command as click on choose a sample combo box then no action is performed.

**Expected Result:**
Combo box should have a clearly defined visual label that is visible to users and when the voice access user gives command as click on code sample combo box then it should be expanded.

**Note:**
Same issue is observed with NVDA screen reader as well.

**User Impact:**
The absence of a visual label for the 'Select an editor theme and Select a code sample' combo box can impact users, especially those who rely on both visual and auditory cues for navigation. It may result in a less accessible and user-friendly experience.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/labels-or-instructions

Have feedback on bugs logged, please tag bug as “A11yRCA” and add your feedback in the comment.

![There is no visually name label is defined for the Choose a sample](https://github.com/microsoft/rushstack-websites/assets/95913748/d5d57741-506c-40d9-bf5d-67a88807dfd8)

![There is no visually name label is defined for the Light Theme](https://github.com/microsoft/rushstack-websites/assets/95913748/1e291660-6fab-41c5-ae50-5f1432d2b514)
